src/SarifWeb/Services/ValidationService.cs

```diff
@@ -59,9 +60,15 @@ public ValidationResponse Validate(ValidationRequest validationRequest)
                         ResultKind.Fail, ResultKind.Informational, ResultKind.NotApplicable, ResultKind.Open, ResultKind.Open, ResultKind.Pass, ResultKind.Review
                     },
                     ConfigurationFilePath = configFilePath,
-                    TargetFileSpecifiers = new string[] { inputFilePath }
+                    TargetFileSpecifiers = new string[] { inputFilePath },
+                    MaxFileSizeInKilobytes = 1024 * 1024 // 1 GB
                 };
 
+                // Ensure that the input file is formatted as pretty-printed JSON.
+                JObject jObject = JObject.Parse(File.ReadAllText(inputFilePath));
```

Comment:
>Parse [](http://example.com/codeflow?start=42&length=5)

Couldn't we simply seek through the first few characters of the file to try to prove that it's pretty-printed? we will find a \n within the first 5 characters, say.

that's clearly our happy-path, right? since no one complained about this previously. so why perform pretty printing in all cases if the majority of usage to date is with pretty printed content.

Reply:
I think it's a good idea to ensure that the entire log is pretty-printed. Imagine some subsections being on a single line, or some other mal-formatting that could result in multiple flags on a single line.